fileserver/apache2.conf

```diff
@@ -0,0 +1,230 @@
+# This is the main Apache server configuration file.  It contains the
+# configuration directives that give the server its instructions.
+# See http://httpd.apache.org/docs/2.4/ for detailed information about
+# the directives and /usr/share/doc/apache2/README.Debian about Debian specific
+# hints.
```

Comment:
If you can turn this into a simple entry under `mods-enabled`, it would be nicer :-)